### PR TITLE
Simplified external test by renaming all test script to test.sh

### PR DESCRIPTION
**Pull request by abhijeetjejurkar**
1. Renamed all -test.sh to test.sh
2. Removed property script in all test.properties files
3. Updated function set_test_info() removing script variable
4. Updated function set_external_custom_test_info() by removing script variable
5. Updated print_entrypoint() function by removing  parameter also modified some code in function. Same done for print_test_script() function

Fixes: #3419

Signed-off-by: Abhijeet Jejurkar <abhijeet.jejurkar2@gmail.com>

### CODE REVIEW

File: external/dockerfile_functions.sh [hunk 1]
```diff
@@ -415,15 +415,12 @@ print_test_results() {
 print_test_script() {
     local file=$1
     local test=$2
-    local script=$3
 
     if [[ ${check_external_custom_test} -eq 1 ]]; then 
         echo -e "# This is the main script to run ${test} tests" \
-                "\nCOPY external_custom/dockerfile/${script} /${script}" \
```
Comment (sophia-guo) [inline external/dockerfile_functions.sh:422]: This shouldn't be removed. It should be updated as `"\nCOPY external_custom/dockerfile/test.sh /test.sh" \`. Same as line 426. Test.sh still need to be copied to docker image.

File: external/dockerfile_functions.sh [hunk 2]
```diff
@@ -472,13 +469,12 @@ print_external_custom_parameters(){
 
 print_entrypoint() {
     local file=$1
-    local script=$2
-    local os=$3
+    local os=$2
 
     if [[ "${os}" = "alpine" ]]; then
-        echo -e "ENTRYPOINT [\"/bin/ash\", \"/${script}\"]" >> ${file}
```
Comment (sophia-guo) [inline external/dockerfile_functions.sh:479]: \"/${script}\"` should be updated to test.sh. Same as line 481

**Comment by sophia-guo** [comment]
As function print_entrypoint() and print_test_script() signature is changed. The way they are called will also need to update. 

https://github.com/adoptium/aqa-tests/blob/master/external/dockerfile_functions.sh#L575-L577
https://github.com/adoptium/aqa-tests/blob/master/external/dockerfile_functions.sh#L586

**Comment by sophia-guo** [comment]
@abhijeetjejurkar could you please resolved the conflicts first so we can run grinder to test the PR. Also I believe function print_test_files() has been updated in another PR so you might need to update it as the function signature changed.

**Comment by abhijeetjejurkar** [comment]
@sophia-guo  Please Tell me if anything more to do. I am new to github tool so I am trying my best to understand git. The Conflict and merge were new for me.

**Comment by sophia-guo** [comment]
Google is also a good helper :-) https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/addressing-merge-conflicts/resolving-a-merge-conflict-on-github

**Comment by abhijeetjejurkar** [comment]
> Google is also a good helper :-) https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/addressing-merge-conflicts/resolving-a-merge-conflict-on-github

Thankyou for Link, Yesterday I Done Youtube, but I guess it had not helped me.

**Comment by sophia-guo** [comment]
grinder https://ci.adoptopenjdk.net/view/work-in-progress/job/grinder_sandbox_new/530/

**Comment by smlambert** [comment]
@abhijeetjejurkar - as with other PRs, please remove the "This Update is Done in AQAvit v0.8.0" statement in your description, as this is being done in master / tip of AQAvit (and will become part of AQAvit v0.9.0 when we create a new release of the project and highly likely _not_ be backported to v0.8.0 which will be obsoleted by v0.9.0).

**Comment by abhijeetjejurkar** [comment]
> @abhijeetjejurkar - as with other PRs, please remove the "This Update is Done in AQAvit v0.8.0" statement in your description, as this is being done in master / tip of AQAvit (and will become part of AQAvit v0.9.0 when we create a new release of the project and highly likely _not_ be backported to v0.8.0 which will be obsoleted by v0.9.0).

Sorry, This was last one. I just removed it.

**Comment by smlambert** [comment]
@sophia-guo - maven issue in the Grinder you kicked off

```
17:39:26  Picked up JAVA_TOOL_OPTIONS: -Dfile.encoding=UTF8 -Djava.security.egd=file:/dev/./urandom
17:39:29  [[1;31mERROR[m] Error executing Maven.
17:39:29  [[1;31mERROR[m] java.lang.IllegalStateException: Unable to load cache item
17:39:29  [[1;31mERROR[m] Caused by: Unable to load cache item
17:39:29  [[1;31mERROR[m] Caused by: Could not initialize class com.google.inject.internal.cglib.core.$MethodWrapper
17:39:29  
17:39:29  netty_test_0_FAILED
```

**Review by sophia-guo — APPROVED** [review]
Maven issue is not related with this one.

**Review by smlambert — APPROVED** [review]
thanks @abhijeetjejurkar !

**Pull request merged**